services/hierarchy/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ group = "org.eclipse.apoapsis.ortserver.services"
 dependencies {
     api(projects.model)
     api(projects.services.authorizationService)
+    api(projects.services.reportStorageService)
```

Comment:
Would it be sufficient to have this as `implementation` dependency?

Reply:
Yes, did not know the difference.